w06-ruby-on-rails/d3-dawn-ruby-oop/README.md

```diff
@@ -0,0 +1,176 @@
+#Ruby <3's OOP
+
+##Learning Objectives
+By the end of lesson, students will be able to...
+
+* Distinguish between objects & classes in Ruby
+* Initialize classes
+* Define attributes & instance methods
+* Distinguish between instance and class variables
+
+##Hashes (10m)
+
+Let's create a new Hash
+
+* Hashes are simple key/value stores.
+
+**Challenge:**
+How can I organize my data using key/value pairs in Ruby? Like so:
```

Comment:
You might consider adding info about 1.8 vs. 2.0 hashes.  The syntax is very different and it's not obvious.  (I white boarded this with my squad and I think it started to help)  Also maybe an example where the key is not a symbol (also tricky).